internal/dao/dao_interfaces.go

```diff
@@ -42,12 +42,15 @@ type PubkeyDao interface {
 
 var GetPubkeyResourceDao func(ctx context.Context) (PubkeyResourceDao, error)
 
-// PubkeyResourceDao TODO
+// PubkeyResourceDao interface for manipulating data of Pubkey deployed into cloud provider.
+// Allows to track the key in the cloud provider and manage it.
+// All methods are unscoped by Account and thus should not get user input as input directly,
+// input should first be validated by PubkeyDao.GetById
 type PubkeyResourceDao interface {
-	GetResourceByProviderType(ctx context.Context, pubkeyId int64, provider models.ProviderType) (*models.PubkeyResource, error)
-	ListByPubkeyId(ctx context.Context, pkId int64) ([]*models.PubkeyResource, error)
-	Create(ctx context.Context, pkr *models.PubkeyResource) error
-	Delete(ctx context.Context, id int64) error
+	UnscopedGetResourceByProviderType(ctx context.Context, pubkeyId int64, provider models.ProviderType) (*models.PubkeyResource, error)
+	UnscopedListByPubkeyId(ctx context.Context, pkId int64) ([]*models.PubkeyResource, error)
+	UnscopedCreate(ctx context.Context, pkr *models.PubkeyResource) error
+	UnscopedDelete(ctx context.Context, id int64) error
 }
```

Comment:
Wow that's quite harsh, I like it! No one will ever miss the fact that a DAO function is not scoped.

Reply:
@adiabramovitch opinions on this? I'm fine suffix or prefix, I just wanted to go with more strict to provoke a though and see what y'all think :)